web/src/lib/stores/settings.ts

```diff
@@ -0,0 +1,15 @@
+import { localWritable } from '$lib/store';
+
+export interface Settings {
+	apiBaseurl: string;
+}
+
+export const settingsStore = localWritable('settings')<Settings>({
+	// apiBaseurl: 'https://api.salz.life',
```

Comment:
unnecessary comment?

Reply:
Ahh I ended up committing it. That should be the default and it's why it's there, but at one point I commented it out so that I can try hooking up to localhost instead.